config/locales/en.yml

```
@@ -139,12 +139,12 @@ en:
       page:
         answer_type: The answer will be checked to make sure it’s in the selected format.
         hint_text:
-          address: You could use hint text to tell people how you will use their address. For example, ‘We’ll send your licence to this address.’
+          address: You could use hint text to tell people how you'll use their address. For example, ‘We’ll send your licence to this address.’
```

Could use a curly quote here:
```suggestion
          address: You could use hint text to tell people how you’ll use their address. For example, ‘We’ll send your licence to this address.’
```